R/htmlwidgets.R

```diff
@@ -329,16 +333,18 @@ createWidget <- function(name,
 #' @name htmlwidgets-shiny
 #'
 #' @export
-shinyWidgetOutput <- function(outputId, name, width, height, package = name) {
+shinyWidgetOutput <- function(outputId, name, width, height, inline = FALSE,
+                              package = name) {
```

Comment:
Maybe the new arg should be last, for backward compatibility? Or do we think literally nobody is specifying the package?